source/common/tracing/zipkin/span_context.h

```diff
@@ -17,18 +18,20 @@ class SpanContext {
   /**
    * Default constructor. Creates an empty context.
```

Comment:
if empty, please default to false if you must default. I'd hate to see trace id 0 accidentally sent to a zipkin endpoint due to a code error. False already says "don't send this"